larray/core/group.py

```diff
@@ -446,7 +449,7 @@ def _to_ticks(s, parse_single_int=False):
 _axis_name_pattern = re.compile(r'\s*(([A-Za-z0-9]\w*)(\.i)?\s*\[)?(.*)')
 
 
-def _seq_str_to_seq(s, stack_depth=1, parse_single_int=False):
+def _seq_str_to_seq(s, stack_depth=1, parse_single_int=False) -> Union[slice, List[Scalar], range, Scalar, None]:
```

Comment:
I don't see any case where this would return None?

Reply:
last line is:
```python
return _parse_bound(s, stack_depth + 1, parse_int=parse_single_int)
```
and in `_parse_bound()`, you can read:
```python
    s = s.strip()
    if s == '':
        return None
```

Reply:
ok. As a side note, I wonder if '' should be accepted by the function, but that's not for this PR.